Patch author:  Orson Teodoro <orsonteodoro@hotmail.com>
Date:  Fri Feb 11 07:09:44 PM PST 2022 (Unix time:  1644635384)
Patch status:  Complete

License header for patched file:

// Copyright 2014 The Souper Authors. All rights reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

--- souper-69536e134478ae1d44c912c90c3db96ad06437c1/include/klee/Config/config.h.orig	2021-06-09 12:16:44.000000000 -0700
+++ souper-69536e134478ae1d44c912c90c3db96ad06437c1/include/klee/Config/config.h	2022-02-09 00:21:07.726437719 -0800
@@ -12,5 +12,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#define LLVM_VERSION_MAJOR 12
-#define LLVM_VERSION_MINOR 0
+//#define LLVM_VERSION_MAJOR 12		// oteodoro:  added via cpp flags for multislot llvm
+//#define LLVM_VERSION_MINOR 0			// oteodoro:  added via cpp flags for multislot llvm
